docs/components/operate/operate-introduction.md

```diff
@@ -18,14 +18,21 @@ dark: useBaseUrl('img/operate-introduction_dark.png'),
 />;
 -->
 
-In addition to providing visibility into active and completed process instances, Operate also makes it possible to carry out key operations such as resolving [incidents](./userguide/resolve-incidents-update-variables.md), and updating process instance variables.
+<!--- is there a reason why we have the comment above? --->
```

Comment:
We can remove this. If you really dig into the git history, it looks like this is an artifact from the original Operate docs that included an image that could toggle for light/dark mode. A relic of the past.